src/modules/course-list/components/pagination/Pagination.component.tsx

```diff
@@ -6,13 +6,13 @@ import { FIRST_PAGE, DEFAULT_ITEMS_PER_PAGE, PAGE_RANGE } from './constants';
 
 import './Pagination.scss';
 
-interface PaginationProps {
+interface Props {
```

Comment:
Same problem as in CourseCard
https://stackoverflow.com/questions/65638991/interface-naming-convention-interface-name-clashes-with-component-name